Palo Alto Cortex Xpanse CCP Connector

v-rajmalviya opened the pull request:
CCF Files

Required items, please complete

Change(s):

Updated Connector Definition & Readme files.
Reason for Change(s):

The changes have already been merged into GitHub.
Version Updated:

Yes
Testing Completed:

Yes
[PaloAltoExpanse_CheckList.xlsx](https://github.com/user-attachments/files/20831427/PaloAltoExpanse_CheckList.xlsx)
<img width="875" alt="image" src="https://github.com/user-attachments/assets/a1382ff4-89dd-4deb-b2a6-07ba9428c340" />

<img width="294" alt="image" src="https://github.com/user-attachments/assets/36abf564-1361-4e72-8211-f8d4a5c9b038" />

<img width="948" alt="image" src="https://github.com/user-attachments/assets/8c73061f-23ed-4264-9fec-540509e95d01" />

v-rajmalviya committed:
Palo Alto Xpanse connector chagnes

Palo Alto Xpanse connector changes

v-rajmalviya committed:
Assets Api related changes removed

Assets Api related changes removed

v-rajmalviya committed:
Merge branch 'master' into v-rajmalviya-PaloAltoXpanse

v-rajmalviya committed:
Update DataConnectorDefinition.json

v-rajmalviya committed:
Update DataConnectorDefinition.json

v-rajmalviya committed:
-

v-rajmalviya committed:
Merge branch 'master' into v-rajmalviya-PaloAltoXpanse

v-rajmalviya committed:
Description changed

Description changed

v-rajmalviya committed:
Code review comments incorporated

Code review comments incorporated

manishkumar1991 reviewed (dismissed):
Approved

manishkumar1991 commented:
![image](https://github.com/user-attachments/assets/3f46e5cd-80f0-484b-8ac3-91b58306f13d)
Naming convention kindly check

v-rajmalviya committed:
Changed Naming of files

Changed Naming of files

manishkumar1991 reviewed (dismissed):
approved

v-prasadboke committed:
Rename CCP to CCF for Cortex Xpanse solution

Renamed all files and references from 'Palo Alto Cortex Xpanse CCP' to 'Palo Alto Cortex Xpanse CCF' to reflect the transition from Codeless Connector Platform to Codeless Connector Framework. Updated template variables, descriptions, and identifiers accordingly. Also updated parameterization in DataConnectorDefinition and PollingConfig for consistency.

v-prasadboke committed:
Rename CCP to CCF in Palo Alto Cortex Xpanse solution

Updated all references from 'CCP' to 'CCF' in createUiDefinition.json, mainTemplate.json, and documentation. Removed obsolete 3.0.4.zip package and updated 3.0.0.zip. Release notes were revised to reflect the initial release under the new naming.

v-maheshbh committed:
solution packaged
